### ISSUE TITLE
[202305][DNX] Orchagent crash when deleting an egress ACL attached to a port-channel

### ISSUE THREAD

**Issue by arista-nwolfe** [sonic-net/sonic-buildimage#17766]
On 202305 I see orchagent crashes when deleting an egress ACL attached to a port-channel on DNX platforms.

Steps to reproduce:
```
config acl add table DATA_EGRESS_IPV4_TEST L3 -s egress -p PortChannel102,PortChannel106,Ethernet76,Ethernet36
config acl remove table DATA_EGRESS_IPV4_TEST
```

I see the following logs:
```
Jan 11 18:35:23.187440 nfc407-5 NOTICE swss#orchagent: :- bindAclTable: Unbind table DATA_EGRESS_IPV4_TEST to ports
Jan 11 18:35:23.187691 nfc407-5 INFO caclmgrd[22395]: ACL change detected for namespace ''
Jan 11 18:35:23.187856 nfc407-5 INFO caclmgrd[22395]: Spawning ACL update thread for namepsace '' ...
Jan 11 18:35:23.188745 nfc407-5 NOTICE swss#orchagent: :- unbindRemoveAclTableGroup: Removing port OID 100000000000b ACL table group ID
Jan 11 18:35:23.190300 nfc407-5 NOTICE swss#orchagent: :- unbind: 100000000000b port is unbound from DATA_EGRESS_IPV4_TEST ACL table
Jan 11 18:35:23.191080 nfc407-5 NOTICE swss#orchagent: :- unbindRemoveAclTableGroup: Removing port OID 1000000000015 ACL table group ID
Jan 11 18:35:23.192549 nfc407-5 NOTICE swss#orchagent: :- unbind: 1000000000015 port is unbound from DATA_EGRESS_IPV4_TEST ACL table
Jan 11 18:35:23.193325 nfc407-5 NOTICE swss#orchagent: :- unbindRemoveAclTableGroup: Removing port OID 2000000000cc2 ACL table group ID
Jan 11 18:35:23.193776 nfc407-5 ERR syncd#syncd: [none] SAI_API_ACL:_brcm_sai_acl_obj_unbind:1798 Invalid acl table.
Jan 11 18:35:23.193810 nfc407-5 ERR syncd#syncd: [none] SAI_API_LAG:_brcm_sai_lag_acl_unbind_all:404 ACL obj lag unbind failed with error -5.
Jan 11 18:35:23.193810 nfc407-5 ERR syncd#syncd: [none] SAI_API_LAG:brcm_sai_dnx_set_lag_attribute:310 ACL obj lag unbind failed with error -5.
Jan 11 18:35:23.193810 nfc407-5 ERR syncd#syncd: :- sendApiResponse: api SAI_COMMON_API_SET failed in syncd mode: SAI_STATUS_INVALID_PARAMETER
Jan 11 18:35:23.193895 nfc407-5 ERR syncd#syncd: :- processQuadEvent: VID: oid:0x2000000000cc2 RID: oid:0x200000001
Jan 11 18:35:23.193895 nfc407-5 ERR syncd#syncd: :- processQuadEvent: attr: SAI_LAG_ATTR_EGRESS_ACL: oid:0x0
Jan 11 18:35:23.194178 nfc407-5 ERR swss#orchagent: :- set: set status: SAI_STATUS_INVALID_PARAMETER
Jan 11 18:35:23.194178 nfc407-5 ERR swss#orchagent: :- bindUnbindAclTableGroup: Failed to unbind PortChannel102 to ACL table group 0, rv:-5
Jan 11 18:35:23.194242 nfc407-5 ERR swss#orchagent: :- handleSaiSetStatus: Encountered failure in set operation, exiting orchagent, SAI API: SAI_API_LAG, status: SAI_STATUS_INVALID_PARAMETER
```

As this error is spawned by syncd I've opened a case with broadcom (CS00012331345).

**Comment by kenneth-arista** [comment]
@wenyiz2021 @arlakshm

**Comment by arista-nwolfe** [comment]
This was fixed in SAI 10.1.13 so closing this issue.
